AWSCRTAndroidTestRunner/app/src/main/cpp/native-lib.cpp

```diff
@@ -0,0 +1,31 @@
+#include <jni.h>
+#include <string>
+#include <sstream>
+
+#include <dlfcn.h>
+
+#include <android/log.h>
+
+typedef int(test_fn_t)(int, char**);
+
+extern "C" JNIEXPORT jint JNICALL
+Java_software_amazon_awssdk_crt_awscrtandroidtestrunner_NativeTestFixture_runTest(
+        JNIEnv *env,
+        jobject /* this */,
+        jstring jni_name) {
+    const char *test_name = env->GetStringUTFChars(jni_name, nullptr);
+    __android_log_print(ANDROID_LOG_INFO, "native-test", "RUNNING %s", test_name);
+
+    test_fn_t *test_fn = (test_fn_t*)dlsym(RTLD_DEFAULT, test_name);
+    if (!test_fn) {
+        __android_log_print(ANDROID_LOG_WARN, "native-test", "%s NOT FOUND", test_name);
+        return 0;
```

Comment:
do we want to exit 0 or something else here?

Reply:
Right now, no, because this is how I'm allowing things like not compiling the log tests, but still having them in the test list work out. Over time, we could fix this and make it a failure, but until we have a clean 1:1 mapping, I don't want to generate tons of failures. Maybe what I should do is come back after all libs are instrumented and make this robust.